validator/client/aggregate.go

```diff
@@ -66,10 +68,16 @@ func (v *validator) SubmitAggregateAndProof(ctx context.Context, slot uint64, pu
 		SlotSignature:  slotSig,
 	})
 	if err != nil {
-		log.WithField("slot", slot).Errorf("Could not submit slot signature to beacon node: %v", err)
-		if v.emitAccountMetrics {
-			ValidatorAggFailVec.WithLabelValues(fmtKey).Inc()
+		status, ok := status.FromError(err)
+		if ok && status.Code() == codes.NotFound {
+			log.WithField("slot", slot).Warnf("Could not find attestation to submit: %v", err)
```

Comment:
What do you think about this?
```suggestion
			log.WithField("slot", slot).Warnf("No attestations to aggregate: %v", err)
```

Reply:
love it, thanks @prestonvanloon !